java/src/apps/ActionListCoreBundle_da.properties

```diff
@@ -33,7 +33,7 @@ jmri.jmrit.MemoryFrameAction = \u00c5ben Hukommelses Monitor
 # services
 jmri.jmrit.withrottle.WiThrottleCreationAction = Start WiThrottle Server
 jmri.jmris.json.JsonServerAction = Start JMRI JSON Server
-jmri.jmris.simpleserver.SimpleServerAction = Start Simpel JMRI Server
+jmri.jmris.simpleserver.SimpleServerAction = Start JMRI SimpleProgAction Server
```

Comment:
`SimpleProgAction` here might be a cut&paste issue

Reply:
Thank you for catching that.